jenkins-slaves/packer-smee-client.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file is not needed anymore, right?

Reply:
I would keep for testing purposes, I mean it is easier to build things "locally" thanks to packer. WDYT?

Reply:
yep, it is, just comment there is is for that purpose

Reply:
`"_comment": "It's not originally created to be used from jenkins but for testing purposes (to build ansible/smee-client.yml in an easier way).",` added

Reply:
I see, ok!